src/arkode/arkode_erkstep_io.c

```diff
@@ -45,8 +45,6 @@ int ERKStepSetErrFile(void *arkode_mem, FILE *errfp) {
   return(arkSetErrFile(arkode_mem, errfp)); }
 int ERKStepSetUserData(void *arkode_mem, void *user_data) {
   return(arkSetUserData(arkode_mem, user_data)); }
-int ERKStepSetDiagnostics(void *arkode_mem, FILE *diagfp) {
-  return(arkSetDiagnostics(arkode_mem, diagfp)); }
```

Comment:
Remove `arkSetDiagonstics` in `arkode_io.c` and remove `report` and `diagfp` variables from `arkode_impl.h`